test/extensions/repository_test.rb

```diff
@@ -112,15 +112,21 @@ def test_search_by_repository_ids
     end
 
     def test_create_or_update_schedule
-      response = @extension.create_or_update_schedule(RepositorySupport.repo_id, 'yum_importer', '2012-09-25T20:44:00Z/P7D')
+      response = @extension.create_or_update_schedule(RepositorySupport.repo_id,
+                                                      'yum_importer',
+                                                      '2012-09-25T20:44:00Z/P7D')
```

Comment:
If we are going to align function parameters, I prefer the more hash like declaration method as I find it more readable and inline with hashes:

```
response = @extension.create_or_update_schedule(
  RepositorySupport.repo_id,
  'yum_importer',
  '2012-09-25T20:44:00Z/P7D'
)
```

Is Rubocop enforcing just the alignment of parameters wherever they are or enforcing that they start from the point of where the `(` column ends?